ansible/roles/openondemand/README.md

```diff
@@ -46,8 +46,16 @@ No other authentication options are required for this method.
 
 ### SSL Certificates
 This role enables SSL on the Open Ondemand server, using the following self-signed certificate & key which are autogenerated by the `mod_ssl` package installed as part of the `ondemand-apache` package. Replace with your own keys if required.
-- `openondemand_ssl_cert`: Optional. Default `/etc/pki/tls/certs/localhost.crt`.
-- `openondemand_ssl_cert_key`: Optional. Default `/etc/pki/tls/private/localhost.key`
+- `openondemand_ssl_cert`: Optional. Default `/etc/pki/tls/certs/localhost.crt` (unless `openondemand_certbot` is true).
+- `openondemand_ssl_cert_key`: Optional. Default `/etc/pki/tls/private/localhost.key` (unless `openondemand_certbot` is true).
+
+Alternatively, you can generate a certificate from Let's Encrypt automatically by configuring the following variables:
+- `openondemand_certbot`: Optional. Default is false. Set to true to request a certificate from Let's Encrypt.
+- `openondemand_certbot_email`: Optional. Default is empty. Set to the admin email address if using Let's Encrypt.
```

Comment:
Does the domain for this have to match e.g. the cluster_domain_suffix or anything?